src/PhotometryFit.cc

```diff
@@ -68,7 +67,7 @@ void PhotometryFit::LSDerivatives(const CcdImage &ccdImage, TripletList &triplet
         // tweak the measurement errors
         double sigma = measuredStar.eflux;
 #ifdef FUTURE
-        TweakPhotomMeasurementErrors(inPos, measuredStar, _posError);
+        TweakPhotomMeasurementErrors(inPos, measuredStar, _fluxError);
```

Comment:
There's no `_posError` in PhotometryFit, but there probably will be `_fluxError` in the future (it is currently unused, thus removed). I'm only removing the preprocessor blocks when I'm sure I won't need to reference them, and cleaning them up as I would other code.